apps/sms/test/marionette/generators/thread.js

```diff
@@ -0,0 +1,54 @@
+function ThreadGenerator() {};
+
+ThreadGenerator.uniqueThreadId = 0;
+ThreadGenerator.uniqueMessageId = 0;
+
+ThreadGenerator.injectDependencies = function(client) {
```

Comment:
I believe we agreed to not generalize so much and remove this ```injectDependencies```, it looks like an alien here :)